kitsune/sumo/static/sumo/js/aaq.js

```diff
@@ -147,15 +149,17 @@
     },
     isDesktopFF: function() {
       // Is the question for FF on the desktop?
-      return document.location.pathname.indexOf("desktop") >= 0;
+      return document.location.pathname.indexOf("desktop") >= 0 ||
+        document.location.pathname.indexOf("firefox-enterprise") >= 0;
```

Comment:
I don't have any objections about the code but I am not sure if we want this here. I would suggest to remove this from here to unblock this PR and open a new one with only this change so that we can discuss it with the rest of the team.